source/options.css

```diff
@@ -155,3 +155,14 @@ li[data-validation] {
 #debugging p:last-child {
 	margin-bottom: 0;
 }
+
+.show-screenshots-label {
+	float: right;
+}
+
+img {
+	max-width: 100%;
+	margin-top: 0.5em;
+	border: 1px solid #d1d5da;
```

Comment:
This uses the same border color as `<kbd>`, it works great on both Chrome's light and dark mode. As for the `border-radius` I choose the same number as `.feature-name`,